contrib/qubes/src/qubes_fwupdmgr.py

```
@@ -423,6 +446,128 @@ def _install_dom0_firmware_downgrade(self, arch_path):
+        allow_reinstall - pass --allow-reinstall to fwupdmgr
+        force - pass -force to fwupdmgr
+        """
+        cmd_install = [FWUPDMGR]
+        if allow_older:
+            cmd_install.append("--allow-older")
+        if allow_reinstall:
+            cmd_install.append("--allow-reinstall")
+        if force:
+            cmd_install.append("--force")
+        cmd_install += ["install", arch_path]
+        p = subprocess.Popen(cmd_install)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)